### DISCUSSION TITLE
Who won Best Actor 🎥🏆 in 2020?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#811] [Galaxy Trivia] [answered]
**Bonus:** For a role in which movie?

**Comment by ParanoidUser** [comment]
@hasanerzi / @devSeungGwan, any ideas? 😶‍🌫️

**Comment by peterbaccaro** [comment, marked as answer]
The Best Actor award for the year 2020 was won by **Joaquin Phoenix** for his role in the film "Joker".
Joaquin Phoenix played the role of **Arthur Fleck/Joker**. The film delves into the origins and transformation of the character, exploring his descent into madness and his emergence as a symbol of chaos.

**Comment by SANKARAMDAS** [comment]
Joaquin Phoenix won best actor at the 2020 Oscars on Sunday for his role as Arthur Fleck in 'Joker.'

**Comment by hasanerzi** [comment]
The Best Actor award for the year 2020 was won by Joaquin Phoenix for his exceptional performance in the movie "Joker."
